app/src/main/java/app/grapheneos/camera/ui/SettingsDialog.kt

```diff
@@ -324,11 +327,33 @@ class SettingsDialog(val mActivity: MainActivity) :
 
         includeAudioToggle = binding.includeAudioSwitch
         includeAudioToggle.setOnClickListener {
+            if (mActivity.videoCapturer.isRecording) {
+                if (ActivityCompat.checkSelfPermission(mActivity, Manifest.permission.RECORD_AUDIO)
+                    != PackageManager.PERMISSION_GRANTED) {
+
+                    // Inform the user why enabling this option isn't possible
+                    mActivity.showMessage("Unable to request for audio permission in between a recording")
```

Comment:
UI strings should be specified via string resources

Reply:
Have pushed a commit to the branch to extract the string into strings.xml